src/Sulu/Bundle/SecurityBundle/Entity/BaseRole.php

```diff
@@ -57,6 +57,20 @@
     private $securityType;
 
     /**
+     * Update the timestamps (changed + created).
+     * This method is mapped to the PRE_PERSIST and PRE_UPDATE
+     * lifecycle events.
+     */
+    public function updateTimestamps()
```

Comment:
I guess that's a functionality that could be in every entity, so it would probably be core stuff. 

I think there is already one trait, but I am not really sure about it... This method is sometimes probably even usable in non-entitiy-classes, that's why I would put it in a trait /cc @chirimoya